### DISCUSSION TITLE
Codigo de java

### DISCUSSION THREAD

**Discussion by 0412Vianey** [4ian/GDevelop#5266] [Archive]
Hola empece a utilizar el apartado de Java en GDevelop pero a la hora de presionar la opción "Vista previa" no carga , no se si me podría ayudar ya revise mi código y esta bien

**Comment by Silver-Streak** [comment]
Hola, tenga en cuenta que no hay soporte para Java en GDevelop. Hay soporte para javascript a través de eventos de javascript, pero espera que usted sea un desarrollador de javascript con experiencia completa y que comprenda la API de javascript. 

Las revisiones del código del juego y la asistencia de gamdev no están disponibles a través de GitHub, pero los enlaces a continuación son para la documentación de la API de JavaScript y los foros de la comunidad de GDevelop. (Traducción del traductor de google)

https://docs.gdevelop.io/GDJS%20Runtime%20Documentation/

https://forum.gdevelop.io/
